domparsing/DOMParser-parseFromString-xml.html

```diff
@@ -50,5 +50,30 @@
     var doc = p.parseFromString("<foo>", type);
     assert_false(doc instanceof XMLDocument, "Should not be XMLDocument");
   }, "XMLDocument interface for incorrectly parsed document with type " + type);
+
+  test(function() {
+    var p = new DOMParser();
+    var doc = p.parseFromString(`
+      <html>
+        <head></head>
+        <body>
+          <script>document.x = 5;<\/script>
+          <noscript><p>test1</p><p>test2</p></noscript>
+        </body>
+      </html>`
+    , type);
+
+    assert_equals(doc.x, undefined, "script must not be executed on the inner document");
+    assert_equals(document.x, undefined, "script must not be executed on the outer document");
+
+    console.log(doc.documentElement.outerHTML);
```

Comment:
```
    console.log(doc.documentElement.outerHTML);
```
Taskcluster failing because the linter’s flagging that `console()` call. https://community-tc.services.mozilla.com/tasks/QAqRwU-iSwWfxCY8hHs3ZQ/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FQAqRwU-iSwWfxCY8hHs3ZQ%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log